Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +80,31 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            return RemoveItemFromIndex(array, index);
+        }
+        public static int[] RemoveItemFromIndex(int[] array, int index)
+        {
+            if (array == null || array.Length == 0)
+            {
+                return array;
+            }
+            if (index < 0 || index >= array.Length)
+            {
+                return array;
+            }
+
+            int arrayLength = array.Length - 1;
+            int[] newArray = new int[arrayLength];
+            int removedIndex = 0;
+            for (int i = 0; i < arrayLength; i++)
+            {
+                if (i == index)
+                {
+                    removedIndex = 1;
+                }
+                newArray[i] = array[i + removedIndex];
+            }
+            return newArray;
         }
```

Comment:
This is identical to what RemoveAt is supposed to do. Put it in there.

Reply:
So I can't use one method to RemoveAt, RemoveLast and RemoveFirst? I should use almost the same code in all three method?